static/json-schema-viewer/index.html

```diff
@@ -6,7 +6,17 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery-mobile/1.4.5/jquery.mobile.min.css" />
         <link rel="stylesheet" href="styles/json-schema-viewer.min.css" />
-
+        <style>
+            .dropdown-container{
+                display: flex;
+                align-items: center;
+                display: flex;
```

Comment:
Is that repetition of `display` and `align-items` necessary here?

Reply:
Could catch. Might be artifacts from me using vim :)